generator.yaml

```diff
@@ -23,6 +23,9 @@ resources:
           resource: Key
           service_name: kms
           path: Status.ACKResourceMetadata.ARN
+      SubscriptionFilters:
+        custom_field:
+          list_of: PutSubscriptionFilterInput
```

Comment:
We could have `list_of: SubscriptionFilter` but `SubscriptionFilter` has a `CreationTime` field that doesn't make sense for the LogGroup spec